### DISCUSSION TITLE
posts/2024/06/one-step-by-one-step/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#59] [General]
# posts/2024/06/one-step-by-one-step/

https://guozheng.rbind.io/posts/2024/06/one-step-by-one-step/

**Comment by earfanfan** [comment]
将来遇到挫折的时候，如果感到无可奈何、无能为力，记得回来看看你自己的初心。

**Reply by residualsun1** [reply]
这两天还会写另外的想法，因为事后回想，觉得原文有些地方其实不够谨慎——对“我想看见的”和“我真的看见的”之间的处理应该要更严肃，防止自己太容易对一件事情浪漫化、理想化，最后陷入自我感动（我不喜欢自我感动，但我似乎又很容易陷入自我感动）。

不过不变的还是初心，希望以后也是。

**Reply by earfanfan** [reply]
还没被社会现实和冷漠人心捶打过的年轻人，容易感动或者说容易产生理想的浪漫的想象，是世界上最正常不过的事了。小伙子，你说的初心具体是指撒呢？

**Reply by earfanfan** [reply]
补充一下，我说的初心和被捶打的关系大概是酱紫的，举个例子……

几天前，我牵着阿木狗走在路上，道路很宽，阿木狗突然停下，走在阿木狗侧后方的一个老头子盯着我认真地看了好一会，然后突然开始骂我挡路然后就走了。我当时忽然明白那个老头子看那好一会，可能是在确认我是不是那种好欺负的人。然后依依就说，咱们以后要训练一下自己的眼神、充满杀气，这样就不会被路上那些欺软怕硬的陌生人欺负了。要是换做以前，我也会生气，然后接着气鼓鼓地看路上的陌生人。不过那天忽然明白，如果真得那么做的话，其实就是我被这个世界改变了啊。因为陌生人并不都是坏人，但如果因为受到了一个陌生人的恶意，从此就假定所有陌生人都是坏人，那么自己也会逐渐变成别人眼中欺善怕恶的陌生人的。

如果延伸开来的话，我已经遇到过很多人，不知道以前曾经受到过什么伤害，但是遇到我的时候已经变成了，为了保护自己先要伤害别人的那种人，于是这些曾经的朋友就变成了我要面对的劫难，要费些时间治愈自己才会让自己终于想明白，不想也变成那样，还是要继续保有对别人的信任。

所以说，我看到的你的初心，只是看你描述的文字，能感受到你表达出来的一些很微弱的很容易达成的触动，还有一些美好的愿望。

**Reply by residualsun1** [reply]
> 还没被社会现实和冷漠人心捶打过的年轻人，容易感动或者说容易产生理想的浪漫的想象，是世界上最正常不过的事了。小伙子，你说的初心具体是指撒呢？

我以前从没想过有这样的可能，如果真是这样，或许这种感动也是可贵的。但伴随着成长，只有感动、理想和浪漫还不够，缺乏力量感。

我想的初心可能就是我对自己所学专业的浪漫化想象（比如比身边的人更敏感社会现象，更关心我之外的人，更重视人和人、人与社会以及人与自然的联系）。我也不知道这种对专业的浪漫化想象从何而来，有时候会在同学们抱怨来这个专业什么也没学到的时候感到微微的“愤慨”，或者是想自己的价值在哪里的时候，然后就冒出以上浪漫化的想象。但明明没读几本书，像这样的口号谁都可以喊，需要警惕这是否会成为一种廉价的自我感动。尤其是我最近接触到一位可能与我年纪相仿的年轻人，写了许多关于哲学思考的文章，虽然我不是内行人，但能从文字中能感受到冷静、自持、洞见的力量——原来写作也可以给予人力量感，让阅读者都觉得宁静下来，有种踏实的感觉。思想上的厚度可以经由文字给予人滋养。我很是羡慕，觉得自己如果只是反复地停留在描写和叙述自己的情感是远远不够的，甚至只是一种“自恋”，我应当积极探索更多的方式去看待自己的生活，并寻找更多元、更有深度的写作方式。因为当下写作是我为数不多表达自己专业性的方式。

---

你举的例子让我想到对社会的信任感。在我所了解的文化里，多数人应该都是对世界抱有善意的，但社会不吃这一套。复杂的社会机制运转、技术托起了现代化便捷、文明的生活，但像你举出的这样的小小例子中的信任是我眼中人们可能很容易让渡出去的事物之一——这个过程其实就是你说的“**捶打**”。如果说我们俩讨论的“理想化”、“浪漫”是进入社会前的个体已拥有的，为了适应社会经济、技术、工业等等错综复杂要素的发展，人就得牺牲一些东西，变得所谓“高效”、世故、练达甚至狠心**一些**，这样才能一起维持社会这个超有机体的运转，无可避免。所以教育很重要的一个功能就是让人学会在容易消磨情感的社会生活里怎么看待情感——是继续相信它、保存它甚至想办法把它传递给别人，还是忽视它、丢掉它，让自己更有效率地生活，一身轻松？我的专业老师给的答案是”把身体和情感交还给社会“。但选择只是第一步，下一步是怎么实践。

看过你举的例子，我又想到这是你的自我教育过程，可能是通过自己的阅读，也可能是通过和朋友的交谈，但无论是什么方式，你都选择了思考，而当你在自己的博客中，或在此处用文字表述出来，就成为我眼中你对待你自己初心的方式。我觉得这是可贵的，而我可能是担心自己的写作会停留在安慰的阶段，写自己想看见的东西，写可以给自己安慰的内容，抚慰一下心里的情感，但不是给自己赋能，如此以后真正遭遇挫折时，至多会短暂地陶醉在自己曾经的”善良叙述“中，但在现实中又妥协、让渡出本可以不失去的一些品质。所以我要实践的是警惕自己，先是不满于自己说的一类”写给自己看就好了“（但并非全盘否定）——比如《[有谁共鸣](https://guozheng.rbind.io/posts/2024/05/me-resonance-world/)》这篇和《[身体是灵魂的燃料](https://guozheng.rbind.io/posts/2024/05/body-for-soul/)》这篇，现在再看、再想，自己便觉得满脸通红，很不好意思，觉得就是无病呻吟。

可能这样说有点乱，但分享一个我从袁长庚老师那儿听到的话或许会清晰一些：袁老师的部分学生说他们本来都是二本的学生考上来读研究生，按照当下普遍的学校体制系统要求读完书毕业就好了，为什么袁老师还要带着他们去读一些很难理解、要硬啃的社会学和人类学文献和书籍呢？袁老师回应的大概意思是，我们可以用很多通俗而又富有解释力的话语看待世界、给自己一个解释，大家聚在一起互相安慰，而后暂时得到力量，但这是很廉价的，唯有克服一些困难，真正接触复杂、抽象的知识，才能得到充实心灵的和实在的力量——关于这点我很认同他，也是我对容易产生感动、浪漫和理想化要有疑虑、警惕的原因。

（p.s.感觉自己有时候表述能力不足，会需要发大段大段的话来表达，希望不会给你回复的压力，看过之后完全可以不回复哈哈）

**Reply by earfanfan** [reply]
小伙子，你可以尝试多信任我一点，你这大段文字完全不会给我造成任何压力，因为我给你留言通常也是这样的，总得展开写一大段才感觉能把脑中捕捉到的思绪给表达出来。我对于网络上的交流通常是这样的心态，我认可了自己的“认可欲求”，确实是在我留言的时候是会希望得到反馈的，也就是存在认可欲求，但是如果没有反馈，我也能够调整好，对“没有反馈”尽量去理解和包容吧。哈哈，所以说呀，如果以后我给你留言写一大段，你也不用有任何回复的压力，就放着是完全没关系的，当然也不用担心会给我撒压力。

你前面提到的那个年轻人也是有公开博客的吗，有的话，可不可以把地址发我瞅瞅，被你这样描述后，我感到非常好奇。你提到你写的另外两篇，我还没来得及看呢，哈哈，这周末就抽空来瞅瞅。

”把身体和情感交还给社会“ 这个答案我没看懂，我理解的是，首先还是要融入社会么？然后在社会的浮沉之中，被改变的部分坦然接受，不愿意被改变的部分坚持保留？

从我的视角看你的表达，大概是酱紫的，作为一个年轻小伙子，对世界的感知还很丰富，尚未经历“捶打”，由于被触动能够相对频繁地发生，所以有些担心如此日常的被触动是不是太多了，会不会沉淀得不够，会不会将来面对挫折时还没积攒到足够的力量。我想，即便你因为认为容易被触动而觉得这很廉价，但也还是值得记录下来的，这是你的一部分，今天的你看昨天的记录还会觉得脸红，一年后的你再看也许会觉得稚嫩，再过十一年，到了我的年纪再看，唉，只会觉得那是不可能复现的，是过去的自己的一部分。我觉得，你将被触动记录下来是很好的，将你现在思考对容易被触动的批判记录下来也是很好的。人确实应该要认识到自己有局限、有变化。

最后，其实大家认识自己的过程都是这样的，如同你这篇博客的题目，一步步来吧。

**Reply by earfanfan** [reply]
对了，小伙子你追求的深度和力量，也是需要时间来积累的，一般情况下会是“由量变达成质变”的路线，现在的感受和思考全都是积攒量的过程。似乎，反思、对自己的批判也是前进的动力。人类这么复杂，感性和理性并存也挺好的。按照我的历史经验来说的话，你的这一次“纠结”其实就是你即将找到一个新的答案的过程。

**Reply by residualsun1** [reply]
> 哈哈，所以说呀，如果以后我给你留言写一大段，你也不用有任何回复的压力，就放着是完全没关系的，当然也不用担心会给我撒压力。

好呀！那我就“放纵”地表达了，同样地，你完全放着也毫无关系:D.

> 你前面提到的那个年轻人也是有公开博客的吗，有的话，可不可以把地址发我瞅瞅，被你这样描述后，我感到非常好奇。

其实准确来说我只知道ta也是学生，所以就默认年纪和我相仿了。ta不是在网站上写作，是在公众号，我前两天听了项飙的讲座《[认得和认可：为了承认的挣扎](https://mp.weixin.qq.com/s/Tl2soGx0RkqilSKs8IYtzg)》，想起了德国法兰克福学派的第三代领军人物就提过“承认”的概念，于是在公众号搜索了一下，就搜到了ta的一篇文章《[一代比一代“保”的法兰克福学派](https://mp.weixin.qq.com/s/B-4Tmodc4Uof_PMM3IFp2Q)》，ta将四代法兰克福学派思想家的思想梳理并批判了一遍，这首先就吸引了我（因为我大概也能看懂，认为他写得还是有逻辑，也清晰，而这显然需要大量的阅读和思考），最后还毫不留情地批判了第四代人物罗萨——我恰好在前面的博文《[有谁共鸣：思罗萨共鸣理论](https://guozheng.rbind.io/posts/2024/05/me-resonance-world/)》写到了罗萨提出的共鸣理论。ta说：

> “**法兰克福发展到第四代，只能用无聊二字来形容**。学术领袖弗斯特的政治哲学被概括为“以宽容、辩护、规范为核心的批判的正义理论”（王凤才，2015），可以看出是在霍耐特的道路上走得更远了。我想如果你认同第一代法兰克福学派尖锐的批判意识，你看到这几个字应该就失去了解的欲望。另一个值得一提的是罗萨，他打出了回到异化的口号，但却将异化的根源指认为“社会加速”：科技加速、社会变迁的加速和生活节奏的加速构成了现代人不适状态的根源。而他开出的药方是重建人与自身、人与自然之间的“共鸣”。感兴趣的朋友可以阅读下介绍性小册子《新异化的诞生》，在这里就不作展开了，**因为我个人认为异化不谈体制、不谈内在于社会的结构性问题，就毫无意义**。
>
> 如果要做一个简单的回顾，第一代学者展现出的是具有浪漫主义色彩的反叛精神和革命热情，将批判的矛头指向现代性-理性-资本主义的合谋；哈贝马斯旨在勾勒一个代议制民主的理想范型，试图用沟通交流的力量解决社会问题；霍耐特的哲学是白左的哲学，是身份政治的哲学，所谓承认不过是发达资本主义国家内部的游戏；**到了第四代，只能说是无病呻吟**。

这一面更加让我对ta好奇，一面也让我有些难为情（另，我个人也认可谈异化，不触及内在的社会结构性问题也无益于解决问题。理论很漂亮，甚至挺有诗意，但缺乏力量），因为我才一度推崇过罗萨的理论——这让我想到我的思考其实往往停留在比较浅薄、容易解释的地方，抽象能力还不够，所以很容易为一些相对通俗、看起来又很契合生活现象的说法而吸引（比如我经常攀附项飙、袁长庚常与公众对话的学者的通俗话语，在此基础上写作，却很难深入分析某现象），但要再进一步做出分析是不行的。相比之下，ta对抽象理论的思考和解释让我觉得我们之间在思想上存在不小的差距，我还是太嫩了。之后我就摸进ta的公众号，看到了一系列关于哲学思考的文章，比如ta分析西方马克思主义和马克思主义《[西马非马？现代性批判还是资本批判](https://mp.weixin.qq.com/s/42zL83PMpm8E_m7pvwAOIg)》（ta比较推崇马克思主义，但推崇的是非意识形态工程下的马克思主义），写《[当代激进左翼四剑客](https://mp.weixin.qq.com/s/5M7x6e-Fzq89oWajLj-DeA)》、然后根据《[【万粉总结】回顾、反思与展望](https://mp.weixin.qq.com/s/xHNuqVj3YeegIOAmOcnK7A)》了解到ta从22年开始写文章，自认为“早期文章的理论性不强，内容比较零散，也没有明确的立场”，接着从女性主义开始写，写到马克思、法兰克福学派，如果想了解ta较为系统的思想痕迹，ta也整理好放在了《[【年终总结】有理想、不麻木、爱智慧](https://mp.weixin.qq.com/s/ChWA3BH-grwO28V4kYbYSw)》中。这时候我基本了解ta主要在写自己对各哲学、社科理论的理解，以及对实事的评论，虽然没有一一细细看过，但通过ta引证文献写作、坚持、认真而带有诚意思考的写作，我觉得ta确实给了我某种“思想有厚度可言”的感觉。我最近在看国内的人类学和民族学教材，里面有大量马克思主义的介绍，虽然意识形态很浓，但慢慢去读、去体会**马克思和恩格斯对社会形态演进、国家诞生、经济运作的分析，我觉得这种知识能给人精神上某种力量**，并非强身健体后可以攻击敌人的力量，也不是肉体上的防御，但它似乎能让你在本混沌的生活中变得静、稳，深邃，有种被充实的感觉——我只是朦朦胧胧地感知到什么，但觉得开始能理解袁长庚老师为什么说真正的知识往往抽象而复杂，但能给人力量，因而很佩服这位博主对哲学思考的坚持，以及欣赏ta对马克思主义的看法和解释。

再有，我看了ta早期的文章（ta自认早期文章很零散），比如这篇《[教育、“教育”与自我教育](https://mp.weixin.qq.com/s/cXQhUEG7s7cAZ-SGTCnIjA)》，ta回顾何为教育时，说：

>谈到“教育的本质”这个老生常谈的问题，我们不妨回顾两个经典的观点：柏拉图曾在《理想国》中借苏格拉底之口陈述了**教育并非是将“灵魂里原来没有的知识灌输到灵魂里去”，而是唤醒人内在的“心灵的优点”**，使“灵魂的眼睛”得以直视光明的“善”，艺术和体育在他看来是极其重要的教育手段；卢梭在《爱弥尔》中“教育的本质”一节开头就写道，“**我们栽种草木，让它长成自己的样子**”，他强调在进行社会和公民的教育，先要开展自然的教育：智育、德育、体育和情感教育，一切教育都要遵循人类的本性，使人的身心能够得到自然的发展。

这里面体现的阅读储备以及思维联系能力也让我佩服。然后还提出当代中国教育的职能是竞争与资源分配（阶层流动）、意识形态宣传、学术能力与职业技能训练。

> 关于第二点，我更多感到无奈。对于我国，一元意识形态是必要的。但要真正让每一个学生真正理解马克思主义思想是成本极高的（马克思本人的学养即使是今天人文社科专业的学生大多都难以企及），因此，我们在意识形态教育上只能采取灌输的形式。但灌输的方式就必然会简化思想的复杂性，缺少说服力和现实感。相反，可感易理解的新自由主义思想更能走进年轻人的心里。当自上而下的意识形态教育失去了效力，我们就只能在一望无际的森林中寻找属于我们的人生之树。

这篇《[跳出来：不要“表演悲伤”](https://mp.weixin.qq.com/s/Th5a3q9owoNs9jd2c3TOnQ)》则写出了我时常在心中产生的对自己的疑虑：

> 我并不否认真正的悲伤情绪，但我认为“现代的悲伤=实质悲伤+表演悲伤”，我们在无意识中利用消极情绪达成某些目的，而在这个过程中主体的渲染使悲伤看起来更加沉重。这个逻辑使我们不得不承认，即使只是在极少数的情况中，**这种“伤感”的本质是自负与自恋的**。正如保罗·德曼对卢梭《忏悔录》的解构：**卢梭的忏悔用虚假的欲望代替了真实的欲望，消解了其忏悔的严肃性，其忏悔话语本质上是辩解话语。当我们开始宣泄和使用消极情绪或者刻意将自己放在一个很低的位置的时候，我们一定要问自己一句，你的目的是什么**？
>
> 以上同时也是我对自己的反思。对我来说，避免了这种自我表演式的悲伤后，我们很多时候并没有自己想象的那么难过。而要防止陷入这种循环，最好的方法就是“跳出来”：跳出自我，跳出让自己困扰的事件。这绝不仅仅是像家长教育孩子那样不要以自我为中心，要关心他人的感受。事实上，关心他人感受如果演变为在意他人的看法，本质上还是自恋的。跳出来意味着别把自己太当回事：别把自己的努力太当回事，别把自己的能力太当回事，别把自己做的事情太当回事。我们习惯性地被“我命由我不由天”这样的口号打动，但我们毕竟不是哪吒，老天这么忙，怎么会特别关注具体某个人——真正关心我们的，自始至终只有自作多情的我们自己。

卢梭这个例子一针见血，如果直面自我，我也必须承认，有时候某些写作是为了消解某些应当承受的严肃性，换言之，本应自省而后调整行动的写作变成了安慰自我，最后保护了不愿改变的自我。这些哲学性的思考要求思维全力以赴，以及认真直面自我，去思索、去言说、去分析，这些特点在当下很吸引我。

**Reply by residualsun1** [reply]
> ”把身体和情感交还给社会“ 这个答案我没看懂，我理解的是，首先还是要融入社会么？然后在社会的浮沉之中，被改变的部分坦然接受，不愿意被改变的部分坚持保留？

我的理解其实就是上面说到的“继续相信它（情感）、保存它（情感）甚至想办法把它传递给别人”。结合咱俩之前讨论的语境，具体的例子就是对他人和社会的信任。其实我觉得我的人类学老师也是这个意思，比如在和他人相处时，在社会生活中时，个体能更主动、积极和勇敢地面对自我以外的世界，甚至敞开自己的情感世界，愿意和他人分享自己的感受、看法，更愿意和别人建立联系，同时关心别人的感受，不要淡漠、躲避、封闭、孤立甚至对立。

我的老师很喜欢费孝通，我所看到的他喜欢的点尤其是费孝通说的“将心比心”，望大家都能带着真挚的情感去和他者相处，希冀一个美好的社会能因此而逐渐实现（对他来说，最直接的可能是主动花费更多时间和学生相处，需要在一定程度上将自身从学术评价标准、体系以及行政系统要求中抽离出来——这是需要一定的勇气和魄力的，陪伴学生，倾听学生苦恼，润物无声地引导学生成长。我在去年的九月份集中写过一次我对他“将身体和情感交还给社会”的一次表现《[九月下半旬](https://guozheng.rbind.io/posts/2023/10/late-september-summary/)》，也分享与你。但其实我们学生和他之间的联系远远不止于此）。所以可能和你说的“被改变的部分坦然接受，不愿意被改变的部分坚持保留”不一样。

> 从我的视角看你的表达，大概是酱紫的，作为一个年轻小伙子，对世界的感知还很丰富，尚未经历“捶打”，由于被触动能够相对频繁地发生，所以有些担心如此日常的被触动是不是太多了，会不会沉淀得不够，会不会将来面对挫折时还没积攒到足够的力量。我想，即便你因为认为容易被触动而觉得这很廉价，但也还是值得记录下来的，这是你的一部分，今天的你看昨天的记录还会觉得脸红，一年后的你再看也许会觉得稚嫩，再过十一年，到了我的年纪再看，唉，只会觉得那是不可能复现的，是过去的自己的一部分。我觉得，你将被触动记录下来是很好的，将你现在思考对容易被触动的批判记录下来也是很好的。人确实应该要认识到自己有局限、有变化。

我觉得你讲得容易理解多了，我就有点掉书袋讲黑话了（bushi。我会留着那些记录的，我就是我，还是无可辩驳。

> 现在的感受和思考全都是积攒量的过程。似乎，反思、对自己的批判也是前进的动力。人类这么复杂，感性和理性并存也挺好的。按照我的历史经验来说的话，你的这一次“纠结”其实就是你即将找到一个新的答案的过程。

借你吉言(●'◡'●)！
